docs/projects/even-odd/_posts/2018-12-31-python.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Even-Odd in Python
+layout: default
+last-modified: 2018-12-31
+tags: ['python', 'even-odd', 'syntax']
+authors: "Chase Stevens"
+---
+
+# Even_Odd
+
+Oftentimes when dealing with things and numbers, we want to split up what we're working with into two groups. This property, known as parity, is useful to know. With Python, we can create a function to check the parity of any given integer.
+
+    def even_odd(x):
+        return "Even" if x % 2 == 0 else "Odd"
+
+Here we have a function named 'even_odd' that takes the parameter "x". The function checks to see if x % 2 is equal to 0. If this is true, we return the value "Even", and if it is false, then we return "Odd."
+
+### Modulo
+One symbol which sticks outs is "%". In Python, and in many other programming languages, the symbol % represents a "modulo" operation, which returns the remainder of the first number divided by the second number.
+
+Let's say we wanted to find out the remainder of 5 divided by 2. We'd calculate that 2 goes into 5 twice with a remainder of one. We can express this with the modulo operator as 5 % 2 = 1. Similarly, we could check the modulo of 9 and 3. Since 3 goes into 9 evenly, we would get an answer of 0, expressed as 9 % 3 = 0.
+
+Since an integer is even if it is divisible by two, we can check to see if a number would have any remainder after being divided by two -- or take a number mod two and if that's equal to zero, we know that said number is even.
+
+### Syntactic Sugar
```

Comment:
This section would be a good opportunity to explain that the single line syntax is python's ternary operator. That might help connect with those coming from other languages.